sinja-sequel.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = '>= 2.3.0'
 
-  spec.add_dependency 'sequel', '~> 4.44'
+  spec.add_dependency 'sequel', '~> 5.0.0'
```

Comment:
I don't want to preclude people from running with Sequel 4.49 with deprecation warnings blazing away if they're not ready to go to Sequel 5.0 for some other reason. Can you change this to `'>= 4.49', '< 6'`? Thank you!